test/perf_test/pom.xml

```diff
@@ -25,7 +25,7 @@
     <dependency>
       <groupId>io.confluent</groupId>
       <artifactId>kafka-avro-serializer</artifactId>
-      <version>5.1.2</version>
```

Comment:
FYI. i dont think this will impact CVEs since it is in test folder.  but thanks for the update here.

Reply:
i agree, but good to keep test up to date with the actual pom.xml. thanks!